### PR TITLE
Apply black formatting

### PR DESCRIPTION
**Pull request by oyvindeide**

### CODE REVIEW

File: snapwell/snapecl.py [hunk 1]
```diff
@@ -105,156 +108,168 @@ def activeCellColumn(grid, owc_kw, x, y, z):
...
-
+        raise ValueError(
+            "Could not find cell column for (%.2f, %.2f, %.2f) in grid." % (x, y, z)
+        )
 
     current_swat = -1
     # Called inv_k because it is inv_k = nz - k
     for inv_k in range(0, len(col)):
-        if col[inv_k][3] >= 0:           # x,y,z,a,s
-            current_swat = col[inv_k][4] #       ^ ^
```
Comment (eivindjahren) [inline snapwell/snapecl.py:155]: This comment depends on formatting to make sense so I suggest rewriting it.

**Review by eivindjahren — CHANGES_REQUESTED** [review]

**Comment by eivindjahren** [comment]
It would be good to rebase this to get the new test action and add `black --check` to it while you are at it.

File: snapwell/snapecl.py (hunk 1 above)
Comment (oyvindeide) [reply snapwell/snapecl.py:155]: Sorry for force pushing, but fixed.

**Comment by oyvindeide** [comment]
> It would be good to rebase this to get the new test action and add `black --check` to it while you are at it.

Done, added it as a separate workflow.

**Review by eivindjahren — APPROVED** [review]

**Pull request merged**